src/_cffi_src/openssl/engine.py

```diff
@@ -11,15 +11,24 @@
 TYPES = """
 static const long Cryptography_HAS_ENGINE_CRYPTODEV;
 
+struct rand_meth_st {
+    void (*seed)(const void *, int);
+    int (*bytes)(unsigned char *, int);
+    void (*cleanup)();
+    void (*add)(const void *, int, double);
+    int (*pseudorand)(unsigned char *, int);
+    int (*status)();
+};
```

Comment:
This can be written `typedef struct { stuff } RAND_METHOD;` so you don't need to know the name